lnrpc/rpc.proto

```diff
@@ -1306,6 +1314,24 @@ message PendingChannelsResponse {
     repeated WaitingCloseChannel waiting_close_channels = 5 [ json_name = "waiting_close_channels" ];
 }
 
+message ChannelSubscription {
+}
+
+message ChannelUpdate {
```

Comment:
To prevent confusion with the gossip messages, I think we should rename this. What about `ChannelEvent`?

Reply:
agree, fixed